modules/aerodyn/src/FVW_BiotSavart.f90

```diff
@@ -429,4 +431,176 @@ subroutine ui_quad_n1(CPs, nCPs, P1, P2, P3, P4, Gamm, RegFunction, RegParam, Ui
    !OMP END PARALLEL
 end subroutine  ui_quad_n1
 
+
```

Comment:
On lines 409, 410, 430, and 431 ...
Are those `!OMP` lines supposed to be comments? Or should they be `!$OMP` OpenMP directives that are missing the `$`?